src/ore/src/now.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Now utilities.
+
+use std::convert::TryInto;
+use std::time::{Duration, SystemTime};
+
+use chrono::{DateTime, TimeZone, Utc};
+
+/// A type representing the number of milliseconds since the Unix epoch.
+pub type EpochMillis = u64;
+
+/// A function that returns system or mocked time.
+pub type NowFn = fn() -> EpochMillis;
+
+/// Returns the current system time.
+pub fn system_time() -> EpochMillis {
+    SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .expect("failed to get millis since epoch")
+        .as_millis()
+        .try_into()
+        .expect("current time did not fit into u64")
+}
+
+/// Converts epoch milliseconds to a DateTime.
+pub fn to_datetime(millis: EpochMillis) -> DateTime<Utc> {
```

Comment:
Can you slap a `#[cfg(feature = "chrono")]` on this and mark the chrono dependency as optional but enabled by default?

Reply:
Can you verify I did this correctly? I couldn't call the feature `chrono` because that's the name of a dependency and cargo scolded me.